package.json

```diff
@@ -33,10 +33,13 @@
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "eslint": "^4.19.1",
-    "eslint-config-airbnb": "15.1.0",
+    "eslint-config-airbnb": "16.1.0",
+    "eslint-plugin-import": "^2.12.0",
```

Comment:
`"eslint-plugin-import": "^2.12.0",` is already in the package.json this PR adds duplication